mopidy/install.sh

```diff
@@ -130,15 +121,20 @@ echo
 if [ $EXISTING_CONFIG ]; then
   warning "Resetting $MOPIDY_CONFIG to package defaults."
   inform "Any custom settings have been backed up to $MOPIDY_CONFIG.backup-$DATESTAMP"
-  apt install --reinstall -o Dpkg::Options::="--force-confask,confnew,confmiss" mopidy=$MOPIDY_VERSION > /dev/null 2>&1
+  sudo apt install --reinstall -o Dpkg::Options::="--force-confask,confnew,confmiss" mopidy=$MOPIDY_VERSION > /dev/null 2>&1
   echo
 fi
 
 # Append Pirate Audio specific defaults to mopidy.conf
 # Updated to only change necessary values, as per: https://github.com/pimoroni/pirate-audio/issues/1
 # Updated to *append* config values to mopidy.conf, as per: https://github.com/pimoroni/pirate-audio/issues/1#issuecomment-557556802
 inform "Configuring Mopidy"
-cat <<EOF >> $MOPIDY_CONFIG
+# Reset the config file
+sudo rm $MOPIDY_CONFIG
```

Comment:
I suggest `sudo rm $MOPIDY_CONFIG` here. If the file doesn't exist (e.g. clean install), `rm` will error out.